docs/Customization/Memory-Hierarchy.rst

```diff
@@ -42,18 +60,11 @@ mixins for those too.
     import freechips.rocketchip.subsystem.{WithL1ICacheSets, WithL1DCacheSets, WithL1ICacheWays, WithL1DCacheWays}
 
     class MyL1RocketConfig extends Config(
-        new WithTSI ++
-        new WithNoGPIO ++
-        new WithBootROM ++
-        new WithUART ++
-        new freechips.rocketchip.subsystem.WithNoMMIOPort ++
-        new freechips.rocketchip.subsystem.WithNoSlavePort ++
-        new WithL1ICacheSets(128) ++                              // change rocket I$
-        new WithL1ICacheWays(2) ++                                // change rocket I$
-        new WithL1DCacheSets(128) ++                              // change rocket D$
-        new WithL1DCacheWays(2) ++                                // change rocket D$
-        new freechips.rocketchip.subsystem.WithNSmallCores(1) ++
-        new freechips.rocketchip.system.BaseConfig)
+         new freechips.rocketchip.subsystem.WithL1ICacheSets(128) ++  // change rocket I$
```

Comment:
While the mixins shown are valuable... I don't think this config will work right out of the gate... maybe don't show this in config form... just show the mixins.

Reply:
Why doesn't this config work?

Reply:
I decided to show just the mixins for brevity.